_posts/2019-12-13-use-iaas-object-in-vro.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: Interact with the vRealize Automation IaaS objects on vRealize Orchestrator
+author: aharlaut
+category: VMWare
+tags: vmware vra vro iaas modelmanager javascript
+---
+
+As described on another [blog post](https://vuptime.io/cards/vra-linqpad-explore-iaas-objects/), vRA IaaS objects can be gathered using the model manager.

```

Comment:
Prefer relative URI: `/cards/vra...` instead of `https://...`

Reply:
Done